crates/ruff_linter/src/rules/ruff/rules/mutable_fromkeys_value.rs

```diff
@@ -73,13 +73,7 @@ pub(crate) fn mutable_fromkeys_value(checker: &mut Checker, call: &ast::ExprCall
     if attr != "fromkeys" {
         return;
     }
-    let Some(name_expr) = value.as_name_expr() else {
-        return;
-    };
-    if name_expr.id != "dict" {
-        return;
-    }
-    if !checker.semantic().is_builtin("dict") {
+    if !checker.semantic().match_builtin_expr(value, "dict") {
```

Comment:
Will this generate a syntatically correct fix if we have `builtins.dict.fromkeys()`?

Reply:
I think so

Reply:
I added a test for it in `crates/ruff_linter/resources/test/fixtures/ruff/RUF024.py`, and the snapshot looks fine to me